### DISCUSSION TITLE
Where to report a security vulnerability?

### DISCUSSION THREAD

**Discussion by signed-log** [topgrade-rs/topgrade#479] [Q&A] [answered]
Title says it all

**Comment by SteveLauC** [comment]
Don't know this either, but you can post it in an issue and ignore the issue template.

cc @DottoDev

**Comment by s34m** [comment, marked as answer]
I have enabled the option in the repository settings, so that users now can privately report security vulnerabilities to @SteveLauC or Me.

**Comment by SteveLauC** [comment]
Just bumped `tempfile` to get the issue fixed, seems like we should publish a release to enable @signed-log to package Topgrade for openSUSE

**Reply by signed-log** [reply]
Indeed, a release would allow the update, if that could be done, that would be great !

**Reply by SteveLauC** [reply]
Will do this in next week:)

**Reply by SteveLauC** [reply]
Hi @signed-log, a new version of Topgrade has just been released:)

**Reply by SteveLauC** [reply]
Please don’t package 12.0.1, seems that we have introduced some issues to the system step, see #512
